### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DStatPersonal**
Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### CODE REVIEW

File: vscode_client_2/README.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
-8. Create a new file in your VS Code workspace with the extension .txt
-9. Write your message in the file 
+## Prerequisites
+- Recipient must be connected to a specific mainframe connection that you configure in extension settings.
+- Java version 11
+- Zowe zosmf profile with credentials defined
 
-***NOTE!!!!*** Your message cannot be longer than 80 characters
+## Getting Started
+- Configure mainframe connection and provide details to intended messenger recipient
```
Comment (zeibura) [inline vscode_client_2/README.md:11]: Where?

File: vscode_client_2/README.md [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-2. Make sure you have a Zowe zosmf profile with credentials defined
-3. Open Visual Studio Code and install the Mainframe messenger extension
-4. In extension settings, write the name of your Zowe profile under "zowe profile"
-5. Write your mainframe username and password in "username" and "password"
-6. If you want use a custom name on your messages, write the name under "custom name"
-7. Under "host" write the URL of your mainframe connection in format http(s)://host(port)
-8. Create a new file in your VS Code workspace with the extension .txt
-9. Write your message in the file 
+## Prerequisites
+- Recipient must be connected to a specific mainframe connection that you configure in extension settings.
```
Comment (spyingcyclops) [inline vscode_client_2/README.md:6]: "...to the mainframe connection that you specify..."

File: vscode_client_2/README.md [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
+- Java version 11
+- Zowe zosmf profile with credentials defined
```
Comment (spyingcyclops) [inline vscode_client_2/README.md:8]: z/OSMF

**Review by zeibura — CHANGES_REQUESTED** [review]

**Review by spyingcyclops — COMMENTED** [review]
Brilliant, Dan.

File: vscode_client_2/README.md [hunk 4]
```diff
@@ -1,27 +1,32 @@
...
-11. A box displays at the top of the screen
-12. Write the mainframe username of the person you want to send the message to in the box
+## How to use Mainframe Messenger
 
-**NOTE** The person you send the message to must be logged in to the same mainframe server you put in extension settings
+1. Open Visual Studio Code and install the Mainframe messenger extension
+2. In extension settings, write the name of your Zowe profile under "zowe profile"
+3. Write your mainframe username and password in "username" and "password"
+4. If you want use a custom name on your messages, write the name under "custom name"
+5. Under "host" write the URL of your mainframe connection in format http(s)://host(port)
```
Comment (zeibura) [inline vscode_client_2/README.md:19]: I think these belong in getting started

**Pull request closed without merging**